packages/marko-web-search/config/index.js

```diff
@@ -44,12 +45,24 @@ class MarkoWebSearchConfig {
       assignedToWebsiteSectionIds: Joi.array().items(
         Joi.number().integer().min(1).required(),
       ).default([]),
+
+      contentTypeLabelMap: Joi.array().items(
```

Comment:
Instead, allow each item of `contentTypes` to either be a string (default) or an object with `type` and `label` properties. Then, when mapping these values, if the array item is a string, continue doing the default, otherwise set the id to using the object type and then set the label to what was passed